internal/pkg/admission/mutating-webhook.go

```diff
@@ -60,10 +60,10 @@ func (r *Reconciler) updateMutatingWebhook(ctx context.Context,
 }
 
 func (r *Reconciler) mutatingWebhookConfiguration(
-	clusterAdmissionPolicy *policiesv1alpha2.ClusterAdmissionPolicy,
+	policy v1alpha2.Policy,
 	admissionSecret *corev1.Secret,
 	policyServerName string) *admissionregistrationv1.MutatingWebhookConfiguration {
-	admissionPath := filepath.Join("/validate", clusterAdmissionPolicy.Name)
+	admissionPath := filepath.Join("/validate", policy.GetUniqueName())
```

Comment:
I think is fine to keep `policy.Name()` here, given the policy server is already running in a namespace, this will be always the same for that given policy server.

Same with validating configuration.

Reply:
what if we have two `AdmissionPolicices` with the same name in different namespaces that are attached to the same `PolicyServer`?
In that case the path would be the same if we don't use the unique name

Reply:
You are right! Good catch :)